scripts/convert-databases.bash

```diff
@@ -0,0 +1,231 @@
+            force=true
+            shift
+            ;;
+        --skip-existing)
+            skip_existing=true
+            shift
+            ;;
+        --help)
+            printUsage
+            exit 0
+            ;;
+        *)
+            printUsage
+            exit 0
+    esac
+done
+
+if $force && $skip_existing; then
+    echo Error: Cannot use both --force and --skipexisting
+    printUsage
+    exit 1
+fi
+
+if ! [ -e "$dbconv" ]; then
+    echo Error: Invalid dbconv binary path: "$dbconv" does not exist
+    exit 1
+fi
+
+if ! [ -n "$dst" ]; then
+    echo Error: Missing destination directory \(\-\-dst\)
+    printUsage
+    exit 1
+fi
+
+if ! [ -d "$src" ]; then
+    echo Error: Invalid source directory: \""$src"\" is missing
+    exit 1
+fi
+
+src=$(realpath $src)
+
+if ! [ -d "$src"/l2chaindata ]; then
+    echo Error: Invalid source directory: \""$src"/l2chaindata\" is missing
+    exit 1
+fi
+
+if ! [ -d $src/l2chaindata/ancient ]; then
+    echo Error: Invalid source directory: \""$src"/l2chaindata/ancient\" is missing
+    exit 1
+fi
+
+if ! [ -d "$src"/arbitrumdata ]; then
+    echo Error: Invalid source directory: missing "$src/arbitrumdata" directory
+    exit 1
+fi
+
+if [ -e "$dst" ] && ! $skip_existing; then
+    if $force; then
+        echo == Warning! Destination already exists, --force is set, this will remove all files under path: "$dst"
+        read -p "are you sure? [y/n]" -n 1 response
+        echo
+        if [[ $response == "y" ]] || [[ $response == "Y" ]]; then
+            (set -x; rm -r "$dst" || exit 1)
+        else
+            exit 0
+        fi
+    else
+        echo Error: invalid destination path: "$dst" already exists
+        exit 1
+    fi
+fi
+
+convert_result=
+convert () {
+    srcdir=$(echo $src/$1 | tr -s /)
+    dstdir=$(echo $dst/$1 | tr -s /)
+    if ! [ -e $dstdir ]; then
+        echo "== Converting $1 db"
+        cmd="$dbconv --src.db-engine=leveldb --src.data $srcdir --dst.db-engine=pebble --dst.data $dstdir --convert --compact"
+        echo $cmd
+        $cmd
+        if [ $? -ne 0 ]; then
+            convert_result="FAILED"
+            return 1
+        fi
+        convert_result="converted"
+        return 0
+    else
+        if $skip_existing; then
+            echo "== Note: $dstdir directory already exists, skipping conversion (--skip-existing flag is set)"
+            convert_result="skipped"
+            return 0
+        else
+            convert_result="FAILED ($dstdir already exists)"
+            return 1
+        fi
+    fi
+}
+
+convert "l2chaindata"
+res=$?
+l2chaindata_status=$convert_result
+if [ $res -ne 0 ]; then
+    printStatus
+    exit 1
+fi
+
+if ! [ -e $dst/l2chaindata/ancient ]; then
+    ancient_src=$(echo $src/l2chaindata/ancient | tr -s /)
+    ancient_dst=$(echo $dst/l2chaindata/ | tr -s /)
+    echo "== Copying l2chaindata ancients"
+    cmd="cp -r $ancient_src $ancient_dst"
+    echo $cmd
+    $cmd
+    if [ $? -ne 0 ]; then
```

Comment:
add an option, on by default, that in case some component failed deletes the entire dest-dir.
in this case it should print a message explaining it's deleting dest dir and also has instructions how to disable this behavior.

Reply:
We also want an option that doesn't delete the entire dest dir but only the database that failed (so we can re-run with skip-existing).
Default: delete all
Easy-ish to set: delete only failed
Hard to set: delete nothing

Reply:
I added `--clean` flag: 
https://github.com/OffchainLabs/nitro/blob/1d69881630f86d37ef1a3da60b535c4677cffeb5/scripts/convert-databases.bash#L44-L47